client/js/socket-events/msg.js

```diff
@@ -166,7 +168,7 @@ function notifyMessage(targetId, channel, msg) {
 							badge: "img/logo-64.png",
 							icon: "img/touch-icon-192x192.png",
 							body: body,
-							timestamp: msg.time,
+							timestamp: timestamp,
```

Comment:
So we want to explicitly format this rather than jsut relying on the toString?

Reply:
`msg.time` is a string, not `Date`.

Reply:
Sure, but you've turned it into a date.

Reply:
`Date.parse` returns int.

Reply:
Oh. Duh.